.github/workflows/merge_queue_dco_mock.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Can you add a comment to this file with the same point you make in the PR that this can be removed if https://github.com/dcoapp/app/issues/199 is fixed?